Lesson26/app/feyza/script.js

```diff
@@ -0,0 +1,38 @@
+// Register on Unsplash to get your own API key
+const API_KEY = 'tmNxD4LLLqpyI4nofipBwzD-Z79nZ-IRKOW2W6sQwCs';
+
+// TODO: Add a new button in HTML to clear the screen
+
+
+const clearButton = document.getElementById("clear-button");
+clearButton.addEventListener("click", function(){
+});
+const imageCountSlider = document.getElementById("imageCount");
+const sliderValueDisplay = document.getElementById("sliderValue");
+
+function updateSliderValue(value) {
+  sliderValueDisplay.textContent = value;
+}
+
+const fetchButton = document.getElementById('fetch-button');
+fetchButton.addEventListener('click', function(){
```

Comment:
When the fetch button is clicked, get value from the input field (element with id 'imgCount'). Check that it contains a positive number, then use it in the url below instead of the hardcoded 10 as value of 'count' query param.